src/test/java/edu/ksu/canvas/attendance/services/CanvasApiWrapperServiceUTest.java

```diff
@@ -1,14 +1,19 @@
 package edu.ksu.canvas.attendance.services;
 
 import edu.ksu.canvas.CanvasApiFactory;
+import edu.ksu.canvas.interfaces.AssignmentOverrideWriter;
 import edu.ksu.canvas.interfaces.EnrollmentReader;
 import edu.ksu.canvas.model.Enrollment;
 import edu.ksu.canvas.model.Section;
+import edu.ksu.canvas.model.assignment.Assignment;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'edu.ksu.canvas.model.assignment.Assignment'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)